src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSite.java

```diff
@@ -268,7 +287,21 @@ private synchronized FormValidation doVerifySignatureWithCaCertificate() throws
             }
         }
     }
-    
+
+    @Nonnull
+    @Override
+    protected JSONSignatureValidator getJsonSignatureValidator() {
+        if(withoutSignCheck) {
+            return new JSONSignatureValidator(String.format("No any validation validator for %s", getId())) {
+                @Override
+                public FormValidation verifySignature(JSONObject o) throws IOException {
+                    return FormValidation.ok();
+                }
+            };
+        }
+        return super.getJsonSignatureValidator();
+    }
+
     /**

```

Comment:
Note: Introduced in Jenkins-1.588, jenkinsci/jenkins@86ea65ab6fe768e7fb97782d3e20b44494bdba6c